packages/apps/graph/src/devnet/marmalade/token/create-token-id.ts

```diff
@@ -0,0 +1,46 @@
+import type { IAccount } from '@devnet/helper';
+import { Pact, readKeyset } from '@kadena/client';
+import { dirtyReadClient } from '@kadena/client-utils/core';
+import {
+  addKeyset,
+  composePactCommand,
+  execution,
+  setMeta,
+} from '@kadena/client/fp';
+import { dotenv } from '@utils/dotenv';
```

Comment:
I'm usually wrapping this into another object that does the parsing and validation of the env-vars and then re-exporting with proper types, as `dotenv` props are only `strings`

Reply:
If you check our `dotenv` object, we can see that not all the properties are `string`